src/main/java/org/magicdgs/thaplv/Main.java

```diff
@@ -28,56 +28,35 @@
 package org.magicdgs.thaplv;
 
 import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
-import org.broadinstitute.hellbender.exceptions.UserException;
 
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 /**
  * Main class for thaplv
  *
  * @author Daniel Gomez-Sanchez
  */
-public class Main {
+public class Main extends org.broadinstitute.hellbender.Main {
 
-    /**
-     * The packages we wish to include in our command line.
-     */
-    protected static List<String> getPackageList() {
+    /** The packages we wish to include in our command line. */
+    @Override
+    protected List<String> getPackageList() {
         final List<String> packageList = new ArrayList<>();
-        // TODO: add some of the bundle tools like IndexFeatureFile
-        // packageList.add("org.broadinstitute.hellbender");
         packageList.add("org.magicdgs.thaplv.tools");
         return packageList;
     }
 
-    /**
-     * Entry point for thaplv.
-     */
-    public static void main(final String[] args) {
```

Comment:
Follow the implementation from here: https://github.com/broadinstitute/gatk/pull/2246